src/main/java/com/mparticle/kits/ButtonKit.java

```diff
@@ -185,4 +238,25 @@ private void logError(String message, Throwable t) {
     private void throwOnKitCreateError(String message) {
         throw new IllegalArgumentException(message);
     }
+
+    private List<ButtonProductCompatible> parseAsButtonProducts(List<Product> products) {
+        List<ButtonProductCompatible> buttonProducts = new ArrayList<>();
+        for (Product product: products) {
+            buttonProducts.add(parseAsButtonProduct(product));
+        }
+        return buttonProducts;
+    }
+
+    private ButtonProductCompatible parseAsButtonProduct(Product product) {
+        ButtonProduct buttonProduct = new ButtonProduct();
+        if (product == null) return buttonProduct;
+
+        buttonProduct.setName(product.getName());
+        buttonProduct.setId(product.getSku());
+        buttonProduct.setValue((int) (product.getTotalAmount() * 100));
+        buttonProduct.setQuantity((int) product.getQuantity());
+        buttonProduct.setCategories(Collections.singletonList(product.getCategory()));
+        buttonProduct.setAttributes(product.getCustomAttributes());
```

Comment:
@wessmith Thoughts on including the custom attributes? I added them but now I'm wondering if it's possible for PII to be leaked here.

Reply:
Discussed offline: omitting custom attributes